src/modules/Thread/context/hooks/useResendMessageCallback.ts

```diff
@@ -35,107 +35,110 @@ export default function useResendMessageCallback({
   pubSub,
 }: StaticProps): (failedMessage: SendableMessageType) => void {
   return useCallback((failedMessage: SendableMessageType) => {
-    if ((failedMessage as SendableMessageType)?.isResendable) {
```

Comment:
Nothing has been updated in these massive change but just added early return block in line number 38.

Reply:
The changed version looks better. LGTM